plugins/ps/functions

```diff
@@ -194,7 +194,17 @@ ps_scale() {
   local DOKKU_PROCFILE="$DOKKU_ROOT/$APP/DOKKU_PROCFILE"
   shift 1
 
-  [[ ! -e $DOKKU_SCALE_FILE ]] && generate_scale_file "$APP" "$IMAGE_TAG" >/dev/null 2>&1
+  # shellcheck disable=SC2015
+  [[ ! -e $DOKKU_PROCFILE ]] && extract_procfile "$APP" "$IMAGE_TAG" >/dev/null 2>&1 || true
```

Comment:
Can we change this to:

```shell
if [[ ! -e $DOKKU_PROCFILE ]]; then
  extract_procfile "$APP" "$IMAGE_TAG" >/dev/null 2>&1 || true
fi
```

I think I saw a lint error related to this last night, hence the request.

Reply:
I added `# shellcheck disable=SC2015` to fix the lint error. According to the ShellCheck wiki this is the correct thing to do in this case. I'm happy to change the code to an `if` if you still feel like it's useful though.

Reply:
Eh this is fine for now then. At some point I'll rewrite this plugin in Golang, at which point all the shellcheck issues go away :)